### ISSUE TITLE
Installing menoh_onnx_viewer

### ISSUE THREAD

**Issue by msakai** [pfnet-research/menoh#27]
`menoh_onnx_viewer` is a convenient tool for using Menoh.
Is there any reason `make install` not installing `menoh_onnx_viewer` into `bin` directory?

**Comment by okdshin** [comment]
No. It's mere conventional issue. I will modify to install `menoh_onnx_viwer` into `bin`.
